### DISCUSSION TITLE
Chart using bar and line type in the same chart

### DISCUSSION THREAD

**Discussion by itpmsoftware** [orchidsoftware/platform#2560] [Q&A] [answered]
Can I create a chart using two different types?, like example
![image](https://user-images.githubusercontent.com/65979963/225982637-f70c0461-a08b-49a1-b666-173ac610402e.png)

I have this target
![image](https://user-images.githubusercontent.com/65979963/225983935-b31147eb-6e5c-49c9-803a-7afff062d4ae.png)
but I'm not sure if i can do that

**Comment by c-gross** [comment, marked as answer]
Should be possible. The chart framework ist Frappe Charts
See: https://frappe.io/charts/docs/basic/stacked_and_mixed

**Reply by itpmsoftware** [reply]
Thanks you, I don't what is the chart framework.

**Reply by c-gross** [reply]
You can see the used npm packages and versions in the package.json:
https://github.com/orchidsoftware/platform/blob/master/package.json
